src/org/traccar/protocol/OwnTracksProtocolDecoder.java

```diff
@@ -70,34 +114,103 @@ protected Object decode(
             position.setAccuracy(root.getInt("acc"));
         }
         if (root.containsKey("t")) {
-            position.set("t", root.getString("t"));
+            String t = root.getString("t");
```

Comment:
Can you please use better names for variables.

Reply:
ok done (t -> trigger)

Reply:
and other abrreviations (rty, anum)